lang/strings_hungarian.txt

```diff
@@ -911,6 +911,7 @@ $s_start_date_label = 'Kezdődátum:';
 $s_end_date_label = 'Záródátum:';
 $s_use_date_filters = 'Szűrés a beküldés dátuma szerint';
 $s_use_last_updated_date_filters = 'Szűrés a frissítés dátuma szerint';
+$s_use_due_date_filters = 'Szűrés az esedékesség dátuma szerint';
```

Comment:
It's not recommended practice, but it does not hurt either - TranslateWiki will pick it up. It's less work to leave this in, instead of merging the PR without and updating the string on translatewiki.net in a second phase.